framework/sessions/__init__.py

```diff
@@ -77,6 +76,7 @@ def prepare_private_key():
 
 
 def get_session():
+    Session = apps.get_model('osf.Session')
```

Comment:
It does, and is replaced by the same error caused by importing `archiver.listeners`.